src/components/check-your-phone/check-your-phone-controller.ts

```diff
@@ -115,6 +72,104 @@ export function checkYourPhonePost(
   };
 }
 
+async function handleMfaChange(
+  intent: Intent,
+  newPhoneNumber: string,
+  updateInput: UpdateInformationInput,
+  sessionDetails: any,
+  req: Request,
+  service: CheckYourPhoneServiceInterface
+): Promise<boolean> {
+  if (intent === INTENT_CHANGE_PHONE_NUMBER) {
+    return handleChangePhoneNumber(
+      newPhoneNumber,
+      updateInput,
+      sessionDetails,
+      req,
+      service
+    );
+  } else if (intent === INTENT_ADD_MFA_METHOD) {
+    return handleAddMfaMethod(
+      newPhoneNumber,
+      updateInput,
+      sessionDetails,
+      req,
+      service
+    );
+  } else {
+    // TODO: Confirm with UCD which error we should display
+    throw Error(`Unknown phone verification intent ${intent}`);
```

Comment:
This'll show the user a 500 and the generic "Something went wrong" page which I think is the correct one here - users shouldn't be able to get into this state.

Reply:
Fixed.